app/controller/router.py

```diff
@@ -19,8 +19,7 @@
 total_requests = Counter('request_count', 'Total webapp request count')
 list_requests = Counter('list_request_count', 'Total list request count')
 plot_requests = Counter('plot_request_count', 'Total plot request count')
-nltk.download('punkt')
-nltk.download('stopwords')
+
```

Comment:
Still needs these two lines if running without Docker
```suggestion
nltk.download('punkt')
nltk.download('stopwords')
```

Reply:
OK I missed these thing.